apps/estuary/lib/estuary/application.ex

```diff
@@ -41,9 +37,18 @@ defmodule Estuary.Application do
   end
 
   defp validate_topic_exists do
-    case Elsa.Topic.exists?(@elsa_endpoint, @event_stream_topic) do
-      true -> :ok
-      false -> Elsa.Topic.create(@elsa_endpoint, @event_stream_topic)
+    case Elsa.Topic.exists?(
```

Comment:
Personally I think it'd be cleaner to extract these guys into local variables (just for readability).

Reply:
I think the way we've solved that before is by throwing them in functions.

Reply:
Agree^ . This is being done in more than 1 place so they should be in functions